BDArmory/UI/VesselSpawnerWindow.cs

```diff
@@ -595,7 +595,7 @@ private void WindowVesselSpawner(int id)
 
                 if (BDArmorySettings.VESSEL_SPAWN_NUMBER_OF_TEAMS == 0) // FFA
                 {
-                    GUI.Label(SLeftSliderRect(++line), $"{StringUtils.Localize("#LOC_BDArmory_Settings_TournamentVesselsPerHeat")}:  ({(BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT > 0 ? BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT : (BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT == -1 ? "Auto" : "Inf"))})", leftLabel); // Vessels Per Heat
+                    GUI.Label(SLeftSliderRect(++line), $"{StringUtils.Localize("#LOC_BDArmory_Settings_TournamentVesselsPerHeat")}:  ({(BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT > 0 ? BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT.ToString() : (BDArmorySettings.TOURNAMENT_VESSELS_PER_HEAT == -1 ? "Auto" : "Inf"))})", leftLabel); // Vessels Per Heat
```

Comment:
Was this giving a compiler warning or error for you? I don't get any compiler or intellisense warnings about it without the `.ToString()`.

Reply:
yeah, that specific line gave a compile error, for some reason. Probably just my copy of VisualStudio, since the next line is the exact same format, and compile fine, so ???

Reply:
OK, strange. The output (`X` and `Y`) of the two branches of `Q ? X : Y` have to be the same type (or automatically castable to the same type), which I guess VSCode is happy with since the final result is part of a string, but your version of VS isn't.